docs/layouts/shortcodes/cli-installation.md

```diff
@@ -0,0 +1,7 @@
+Install the CLI using this command:
+
+```bash
+curl -fsS https://platform.sh/cli/installer | php
+```
+
+If you run into issues or want to install manually, see the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli#user-content-installation).
```

Comment:
:question: 
Is the different command for Windows not necessary?

Reply:
The part we had in the documentation seemed [fairly outdated](https://github.com/platformsh/platformsh-cli/commit/e9c23686b32e68f28dedb658ef0ae4609c01741d), and since the addition of the `.exe` seems to be a [loose requirement](https://github.com/platformsh/platformsh-cli/commit/7b8588cc7380e5d0389a51d153c3146948aeceff), I thought it might be enough to provide the link to the source of truth.